src/renderers/shared/shared/__tests__/ReactUpdates-test.js

```diff
@@ -1143,25 +1143,14 @@ describe('ReactUpdates', () => {
     expect(mounts).toBe(1);
   });
 
-  it('mounts and unmounts are sync even in a batch', () => {
-    var container1 = document.createElement('div');
-    var container2 = document.createElement('div');
-
-    let called = false;
-    class Foo extends React.Component {
-      componentDidMount() {
-        called = true;
-        ReactDOM.render(<div>Hello</div>, container2);
-        expect(container2.textContent).toEqual('Hello');
-        ReactDOM.unmountComponentAtNode(container2);
-        expect(container2.textContent).toEqual('');
-      }
```

Comment:
You've changed this behavior so you had to update but you didn't replace it with an equivalent test anywhere. This is just less test coverage instead of testing the new behavior.